test/jdk/sun/security/krb5/config/native/TestDynamicStore.java

```diff
@@ -56,6 +56,16 @@ static int action(char what, char whom) throws Exception {
 
     public static void main(String[] args) throws Exception {
 
+        // Show a popup to remind to run this test as sudo user
+        PassFailJFrame.builder()
```

Comment:
Using pass/fail in this instance is a little confusing. 

Because this test runs on mac can you just check the `user.name` property? A quick, simple test shows that when java is run with `sudo`, the property returns "root". This way, the person running the test isn't constantly faced with a dialog box.